zuko-ui/src/components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,72 @@
+.footer {
+    background-color: #FFF8E7;
+    padding: 121px 25px 70px;
+    @media only screen and (min-width: 1024px) {
```

Comment:
Use our mixing for responsive. Use can see our test component to see how we import mixins in scss file